### PR TITLE
TM-1004 missing notification for language change

### PR DESCRIPTION
**Pull request by michalsikora**

### CODE REVIEW

File: src/admin-panel/modules/alert/services/RefreshAlert.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    }
+
+    /**
+     * Add success message after refresh page
+     *
+     * @param localStorageKey
+     * @param message
+     */
+    success (localStorageKey, message) {
```
Comment (MarcinMat) [inline src/admin-panel/modules/alert/services/RefreshAlert.js]: Add tests for this method f.e:
'when calling succes should add a new alert'
'when calling succes should remove item from local storage'

File: src/admin-panel/modules/alert/services/RefreshAlert.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * @name alert#RefreshAlert
+ * @namespace clearcode.tm.alert
+ */
+
+/**
+* @description Remove Alert after Timeout
+*
+* @param {Integer} id from storage
+*/
+var removeAfterTimeout = function (id) {
```
Comment (MarcinMat) [inline src/admin-panel/modules/alert/services/RefreshAlert.js]: Duplicate code from Alert.js.

File: src/admin-panel/app/modules/user/controllers/BaseController.js [hunk 3]
```diff
@@ -57,6 +61,8 @@ class BaseController {
                     .add(translations[breadcrumbText])
                     .broadcast();
 
+                refreshAlert.success('LANGUAGE_CHANGE_MESSAGE', translations['The language has been changed']);
```
Comment (MarcinMat) [inline src/admin-panel/app/modules/user/controllers/BaseController.js]: Translation should be done inside RefreshAlert. The way it is now may be counterintuitive if someone else uses this code (normal Alert translates message text inside addAlert() method).

File: src/admin-panel/modules/alert/services/RefreshAlert.js (hunk 1 above)
Comment (michalsikora) [inline src/admin-panel/modules/alert/services/RefreshAlert.js]: this method is covered already.

**Pull request merged**